tools/feature_matrix.py

```diff
@@ -15,12 +15,12 @@
 
 UNSUPPORTED = 0x7FFFFFFF
 
-# Oldest support browser versions.  These have been set somewhat
-# arbitrarily for now.
+# Oldest support browser versions.  Keep in sync with
+# src/settings.js.
```

Comment:
Please add similar comments in `settings.js` (that would have led me to this file, for example).

Or perhaps we can remove the numbers in this file and just say "see minimum versions in settings.js"?

Reply:
The comments in settings.js are generally for users of the settings, not us developers, so I'm not sure it makes sense to add such comments there.

Sadly we cannot remove these since there are needed at runtime.

Reply:
Hmm, good points. But I am quite worried that we will decide to bump the minimum version of some browser, and only remember to do that in `settings.js`. I, for example, completely forgot about version numbers in this file just now, and didn't even think to search in this directory.

Perhaps `settings.js` could just add `see feature_matrix.py for more` comments?

Reply:
OK done.  PTAL